src/libs/connectorLibs/saveFiles.js

```diff
@@ -272,13 +272,15 @@ async function getFileFromPath(client, entry, options) {
   }
 }
 
-async function createFile(client, entry, options, method, fileId) {
+
+async function createFile(client, entry, options, method, file) {
```

Comment:
Probably out of scope, but it feels weird to have the `file` in last position, especially behind `options` and `method`. I would have put it just after the `client` (and removed the method, see other comment)